UXWWhoWeAre.js

```diff
@@ -1,3 +1,79 @@
+const data = [ 
+  {
+    name: 'Keri',
+    role: 'Co-Organizer of UX Wizards',
+    image: '/assets/keri.jpg',
+    description: 'Organizer of the UX Wizards!',
+    link: 'https://www.linkedin.com/in/kerikeri/'
+  },
+  {
+    name: 'Laura',
+    role: 'Co-Organizer of UX Wizards',
+    image: '/assets/laura.jpg',
+    description: 'Laura built the Who We Are Page!',
+    link: 'https://www.linkedin.com/in/lewilliams89/'
+  },
+  {
+    name: 'Seiko',
+    role: 'Product Designer',
+    image: '/assets/seiko.jpg',
+    description: 'Seiko built the Logos and Icons Page!',
+    link: 'https://www.linkedin.com/in/seikoigi/'
+  },
+  {
+    name: 'Kaisen',
+    role: 'Software',
+    image: '/assets/kaisen.jpg',
+    description: 'Kaisen kept us on track.',
+    link: 'https://www.linkedin.com/in/kaisen-lin-02bb0864/'
+  },
+  {
+    name: 'Javi',
+    role: 'Frontend Developer',
+    image: '/assets/javi.jpg',
+    description: 'Javi built the Design Assets page!',
+    link: 'https://www.linkedin.com/in/javigme/'
+  },
+  {
+    name: 'Max',
+    role: 'Product Designer',
+    image: '/assets/max.jpg',
+    description: 'Max built the Colors page!',
+    link: 'https://www.linkedin.com/in/maxux/'
+  },
+  {
+    name: 'Sidney',
+    role: 'Product Designer',
+    image: '/assets/sidney.jpg',
+    description: 'Sidney built the Typography page!',
+    link: 'https://www.linkedin.com/in/sidneyrhoads/'
+  },
+  {
+    name: 'Dhruv',
+    role: 'Software',
+    image: '/assets/dhruv.jpg',
+    description: 'Dhruv worked on this page!',
+    link: 'https://www.linkedin.com/in/dhruv-madhok-9290a2a3/'
+  },
+]
+
+const titleStyle = {
+  color: "grey"
+}
+
+const columnStyle = {
+  float: "left",

```

Comment:
You might be better off using Flexbox here instead of trying to float it. It seems like it's changing the positioning of things, which is why the "Next: How We Sound" is being pushed up. Is there a reason you need to float it?